### ISSUE TITLE
ext/XS-Typemap: status in core-p7 branch

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#83]
In this ticket we will track our core-p7 progress on tests in directory `ext/XS-Typemap`.

As 2020-07-09 10:20 EDT, we are getting this output from `make test` for the files in this directory.

```
../ext/XS-Typemap/t/Typemap.t ...................................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
No subtests run 
Global symbol "$Is_Dosish" requires explicit package name (did you forget to declare "my $Is_Dosish"?) at ../lib/AnyDBM_File.t line 15.
BEGIN not safe after errors--compilation aborted at ../lib/AnyDBM_File.t line 21.
# Looks like your test exited with 255 before it could output anything.

```
These results were generated on an unthreaded build on linux.  The results might differ significantly if run on a threaded build.  There have been subsequent commits to this branch which may have impacted these results.  YMMV.

**Comment by atoomic** [comment]
fixed by db93b629acb78bdd5f3f5c661873234ca38923af

**Comment by jkeenan** [comment]
> fixed by [db93b62](https://github.com/atoomic/perl/commit/db93b629acb78bdd5f3f5c661873234ca38923af)

@atoomic, I just updated my core-p7 branch ... and I don't see that commit in it.
```
$ gitcurr
core-p7
[perl] 509 $ git status
On branch core-p7
Your branch is up to date with 'origin/core-p7'.

nothing to commit, working tree clean
[perl] 510 $ git describe
v5.32.0-576-g9e5ecda8ae
[perl] 511 $ gitshowf
commit 9e5ecda8ae6aac420410b9256cde7e3486955f91 (HEAD -> core-p7, origin/core-p7)
Author:     Nicolas R <atoomic@cpan.org>
AuthorDate: Wed Jul 8 16:53:45 2020 -0600
Commit:     Nicolas R <atoomic@cpan.org>
CommitDate: Wed Jul 8 16:53:45 2020 -0600

    Net-Ping no indirects
```
Can you double check where you committed it?

Thank you very much.
Jim Keenan

**Comment by atoomic** [comment]
I've not pushed to the perl5 repo but only to this one for now, to avoid the noise & co lol

**Comment by jkeenan** [comment]
> fixed by [db93b62](https://github.com/atoomic/perl/commit/db93b629acb78bdd5f3f5c661873234ca38923af)

Confirmed.  Thanks.
